src/Components/GrantsFinalizeShow.js

```diff
@@ -140,12 +182,40 @@ export default function GrantsFinalizeShow(props) {
             >
               Make a Copy
             </Link>
-            <Link
+            {/* <Link
               className="btn btn-outline-dark"
               to={`/organizations/${currentOrganizationId}/grants/${grant.id}/edit`}
             >
               Edit
-            </Link>
+            </Link> */}
+            <Button onClick={handleShowGrantCopyModal}>Copy</Button>
```

Comment:
Do we want to remove the "Make a Copy" link or merge it with this button? Also, I don't see the copy button on the mockaframe, do you know if it will just be another button next to edit or somewhere else?

Reply:
A button, I think, and we can remove the link.